pose.py

```diff
@@ -44,7 +44,10 @@ def main():
 
     pose = mp_pose.Pose(
         min_detection_confidence=args.min_detection_confidence, min_tracking_confidence=args.min_tracking_confidence)
-    cap = cv2.VideoCapture(args.device)
+    if args.device is None:
```

Comment:
Have you tested it? When I debug it, it is `0` by default and not `None` because I set it in the utils.py.
So the if else branch is not necessary.

Reply:
I did debug it and worked with the flag and without it attempted to connect to `/dev/video0`